Examples/CodePushDemoApp/android/app/src/main/java/com/microsoft/codepushdemoapp/MainActivity.java

```diff
@@ -12,7 +12,7 @@
 
     @Override
     protected String getJSBundleFile() {
-        return CodePush.getBundleUrl("index.android.bundle");
+        return CodePush.getJSBundleFile("index.android.bundle");
```

Comment:
We can probably simplify the demo app by removing the explicit bundle name parameter.